src/main/java/entity/car/Car.java

```diff
@@ -0,0 +1,58 @@
+package entity.car;
+
+import entity.NumberGenerator;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class Car implements Comparable<Car>{
+
+    private static final Random RANDOM_DICE = new Random();
+    private static final int LEAST_RANDOM_VALUE = 4;
+    private static final int MAX_CAR_NAME_LENGTH = 5;
+    private CarPosition position;
+    private final CarName name;
+
+    public Car(String name) {
+        checkLengthOfCarName(name);
+        checkNullOrBlank(name);
+        this.position = new CarPosition(0);
+        this.name = CarName.from(name);
+    }
+
+    public void moveCar(NumberGenerator numberGenerator) {
```

Comment:
```suggestion
    public void move(NumberGenerator numberGenerator) {
```

Car 객체 안의 메서드라 move()로 구현하는 것은 어떨까?